pkg/cloud/vsphere/services/cloudprovider/csi.go

```diff
@@ -27,6 +27,16 @@ import (
 
 // NOTE: the contents of this file are derived from https://github.com/kubernetes-sigs/vsphere-csi-driver/tree/master/manifests/1.14
 
+const (
+	DefaultCSIControllerImage     = "vmware/vsphere-block-csi-driver:v1.0.0"
+	DefaultCSINodeDriverImage     = "vmware/vsphere-block-csi-driver:v1.0.0"
```

Comment:
Should we use v1.0.1 of the CSI images?

Reply:
yes -- this PR is just about setting defaults, I think image updates should happen in a separate PR